src/backup_camera.cpp

```diff
@@ -27,7 +35,8 @@ bool BackupCamera::init_SDL(SDL_Renderer** empty_renderer, SDL_Window** empty_wi
     }
     else
     {
-        *empty_window = SDL_CreateWindow("Video Application", xpos, ypos, screen_width, screen_height, SDL_WINDOW_BORDERLESS);
+        int windowMode = (m_fullscreenFlag ? SDL_WINDOW_FULLSCREEN_DESKTOP : SDL_WINDOW_BORDERLESS);
```

Comment:
Pretty good use of the Ternary Operator. It usually causes confusion in complex cases but I think this is appropriate.

However, I think this would make it a little easier to read:
`int windowMode = (m_fullscreenFlag == true? SDL_WINDOW_FULLSCREEN_DESKTOP : SDL_WINDOW_BORDERLESS);`

^This suggestion is completely up to you